### PR TITLE
Maine Personal Exemption

### LINKED ISSUE #1297: Maine personal exemption
$4,300 per filer, income-tested
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_book_gen_instr_revMay2022.pdf

### PR DESCRIPTION
**Pull request by leogoldman**
Fixes #1297

### CODE REVIEW

**Review by MaxGhenis — COMMENTED** [review]
Try deleting the extra parameters (`phaseout/phaseout_start.yaml`, `phaseout/phaseout_width.yaml`, and `phaseout_start.yaml`) and adding `amount.yaml`.

**Comment by leogoldman** [comment]
Screenshot 2023-02-16 at 5 10 37 PM
The Python black default line length (88) finally caught up to me (causes the first/Lint tests to fail). @k-obenschain in case this helps
VS-Code Settings > "python formatting provider" > Select black > Add argument "-l 79"

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Maine personal exemption amount
+metadata:
+  unit: currency-USD
+  label: Maine Personal Exemption Amount
+  reference:
+    - title: Instructions for Form 1040ME
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml:6]: Could you add the legislative reference for this one too? I'll merge after that.

**Review by MaxGhenis — APPROVED** [review]

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml (hunk 1 above)
Comment (leogoldman) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml:6]: Actually, it looks like the amount has gone up to $4,450 in 2022 and is scheduled to go up to $4,700 in 2023. Does it seem like I'm understanding that correctly? If so, should I go ahead and put in the different period/amounts?
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ta_nov2022_vol32_iss22.pdf
That page links to (2022 and 2023 resp):
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2022.pdf
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2023.pdf

Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml:6]: Yes please do! And also the legal code for the pre-inflation-adjustment amount.

Comment (leogoldman) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml]: I can also add the old phaseout starts, but as far as I can tell those only changed in 2021 and we didn't do that initially to not imply that we had historical coverage?
https://www.mainelegislature.org/legis/statutes/36/title36sec5126-A.html

Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml]: Yeah we haven't yet set a standard here. Filed https://github.com/PolicyEngine/policyengine-us/issues/1805 to do that.

I meant to cite the code without entering the historical value, but this works. Thanks - merging.

**Pull request merged**